.gitignore

```diff
@@ -58,3 +58,6 @@ tests-fuzz/corpus/
 
 ## default data home
 greptimedb_data
+
+# github
+!/.github
```

Comment:
Is it required?

Reply:
Yes, Without it types will ignore .gitgub folder